src/sinks/aws_cloudwatch_metrics.rs

```diff
@@ -132,7 +133,7 @@ impl CloudWatchMetricsSvc {
         client: CloudWatchClient,
         cx: SinkContext,
     ) -> crate::Result<super::VectorSink> {
-        let default_namespace = config.default_namespace.clone();
+        let default_namespace: ImStr = config.default_namespace.clone().into_boxed_str();
```

Comment:
On the topic of `into`, I wonder if this could simply be `.into()`, since you are specifying the type, which will simplify any potential type change.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b38de7663155acd1b4f151ced108a5cd looks like we could use `into`, yep.